website/core/Site.js

```diff
@@ -29,6 +29,14 @@ var Site = React.createClass({
           <meta property="og:url" content="http://draftjs.org/index.html" />
           <meta property="og:description" content="Rich Text Editor Framework for React" />
 
+          <link rel="apple-touch-icon" sizes="180x180" href="/img/apple-touch-icon.png" />
+          <link rel="icon" type="image/png" sizes="32x32" href="/img/favicon-32x32.png" />
+          <link rel="icon" type="image/png" sizes="16x16" href="/img/favicon-16x16.png" />
+          <link rel="manifest" href="/img/manifest.json" />
+          <link rel="mask-icon" href="/img/safari-pinned-tab.svg" color="#5bbad5" />
+          <link rel="shortcut icon" href="/img/favicon.ico" />
+          <meta name="msapplication-config" content="/img/browserconfig.xml" />
+          <meta name="theme-color" content="#ffffff" />
```

Comment:
just a tiny nitpick - would be nice if theme-color were the red background - `#852d2d`

Reply:
good catch, i will update to make the background-color red